src/main/java/uk/gov/fco/documentupload/api/FileController.java

```diff
@@ -125,25 +130,46 @@ public DeferredResult<ResponseEntity<Void>> store(
                         log.info("Virus scan failed for file");
                         output.setResult(ResponseEntity
                                 .status(HttpStatus.UNPROCESSABLE_ENTITY)
-                                .build());
+                                .body(""));
                     } else {
-                        log.info("File is clean, storing");
-                        String id = storageClient.store(merger.merge(uploads));
-                        output.setResult(
-                                ResponseEntity
-                                        .created(builder.path("/files/{id}").build(id))
-                                        .build());
+                        log.info("File is clean, checking image quality");
+                        boolean passedQualityCheck = true;
+                        for (FileUpload upload : uploads) {
+                            if (!ocrService.passesQualityCheck(upload)) {
+                                passedQualityCheck = false;
+                                break;
+                            }
+                        }
+                        if (!passedQualityCheck) {
+                            log.info("Quality check failed for file");
+                            output.setResult(ResponseEntity
+                                    .status(HttpStatus.UNPROCESSABLE_ENTITY)
+                                    .body(""));
```

Comment:
for 422s, let's return 

```
{
  error: "QualityThresholdError",
  message: "Document did not pass quality threshold" 
}
```

or 

```
{
  error: "VirusScanError",
  message: "Document contained a virus" 
}
```

then in the form runner, we can render different messages based on the error. For anyone using older versions of document upload (which don't return res.error), they'll just render the virus message